src/components/loader/loader.stories.mdx

```diff
@@ -36,38 +43,69 @@ This example of the Loader component demonstrates how it will appear as default.
 </Preview>
 
 ### Small
+
 This is an example of a small Loader component.
 
 <Preview>
   <Story name="small">
-    <Loader size='small' />
+    <Loader size="small" />
   </Story>
 </Preview>
 
 ### Large
+
 This is an example of the large Loader component. The larger size is only used when a whole page is loading.
 
 <Preview>
   <Story name="large">
-    <Loader size='large' />
+    <Loader size="large" />
   </Story>
 </Preview>
 
-
 ### Inside Buttons
+
 This is an example of the Loader nested inside of a Button component. When used inside a primary button the colour of the squares will be white.
 To ensure that the correct styling is applied to the `Loader` component when it is nested inside of the `Button` component,
 please remember to pass the `isInsideButton` prop to the `Loader` component.
 
 <Preview>
   <Story name="inside Buttons">
-    <Button buttonType='primary'>
-      <Loader isInsideButton/>
+    <Button buttonType="primary">
+      <Loader isInsideButton />
     </Button>
   </Story>
 </Preview>
 
+This example of the Loader bar component demonstrates how it will appear as default.
```

Comment:
You should follow the same format as everywhere else in the docs for titles, here I would have 

```
## LoaderBar

<Preview>
  <Story name="default bar">
    <LoaderBar mt={2} />
  </Story>
</Preview>

### Small

<Preview>
  <Story name="small bar">
    <LoaderBar size="small" mt={2} />
  </Story>
</Preview>

### Large

<Preview>
  <Story name="large bar">
    <LoaderBar size="large" mt={2} />
  </Story>
</Preview>```